package.json

```diff
@@ -24,6 +22,7 @@
     "select2-bootstrap-css": "^1.4",
     "waypoints": "^4.0",
     "webpack-notifier": "^1.6.0",
+    "eonasdan-bootstrap-datetimepicker": "~3.1.3",
```

Comment:
Can you move it back to the old line, so we have no diff here?

Reply:
Sure!